build.gradle

```diff
@@ -42,6 +42,7 @@ compileTestKotlin {
 }
 
 patchPluginXml {
+    sinceBuild sinceBuildVersion
```

Comment:
Turns out if we don't have this specified, then the gradle plugin just patches the version with the `ideaVersion` build number. So I guess we do need this. Lol, wups.